README.md

```diff
@@ -302,3 +324,11 @@ class Component extends React.Component {
   }
 }
 ```
+
+### Linting
+
+We use ESLint and Stylelint to lint and automatically format code.
+
+- `npm run eslint` Will run ESLint on the entire code base and automatically try to fix all JS and JS like files.
+- `npm run eslint-new` Will run ESLint only on newly added files in the current git branch and automatically try to fix the JS and JS like files.
```

Comment:
instead of "only on newly added files in the current git branch" this should state "only on uncommitted files". OR you could change HEAD to `master` in the command.

Reply:
I've updated HEAD to master